language/Greek/strings.po

```diff
@@ -2,3380 +2,3381 @@
 
 #: src/emu/inpttype.ipp:74
 msgctxt "input-name"
 msgid "P1 Mahjong O"
-msgstr ""
+msgstr "P1 Mahjong O"
 
 #: src/emu/inpttype.ipp:75
 msgctxt "input-name"
 msgid "P1 Mahjong P"
-msgstr ""
+msgstr "P1 Mahjong P"
 
 #: src/emu/inpttype.ipp:76
 msgctxt "input-name"
 msgid "P1 Mahjong Q"
-msgstr ""
+msgstr "P1 Mahjong Q"
 
 #: src/emu/inpttype.ipp:77
 msgctxt "input-name"
 msgid "P1 Mahjong Kan"
-msgstr ""
+msgstr "P1 Mahjong Kan"
 
 #: src/emu/inpttype.ipp:78
 msgctxt "input-name"
 msgid "P1 Mahjong Pon"
-msgstr ""
+msgstr "P1 Mahjong Pon"
 
 #: src/emu/inpttype.ipp:79
 msgctxt "input-name"
 msgid "P1 Mahjong Chi"
-msgstr ""
+msgstr "P1 Mahjong Chi"
 
 #: src/emu/inpttype.ipp:80
 msgctxt "input-name"
 msgid "P1 Mahjong Reach"
-msgstr ""
+msgstr "P1 Mahjong Reach"
 
 #: src/emu/inpttype.ipp:81
 msgctxt "input-name"
 msgid "P1 Mahjong Ron"
-msgstr ""
+msgstr "P1 Mahjong Ron"
 
 #: src/emu/inpttype.ipp:82
 msgctxt "input-name"
 msgid "P1 Mahjong Flip Flop"
-msgstr ""
+msgstr "P1 Mahjong Flip Flop"
 
 #: src/emu/inpttype.ipp:83
 msgctxt "input-name"
 msgid "P1 Mahjong Bet"
-msgstr ""
+msgstr "P1 Mahjong Ποντάρισμα"
 
 #: src/emu/inpttype.ipp:84
 msgctxt "input-name"
 msgid "P1 Mahjong Take Score"
-msgstr ""
+msgstr "P1 Mahjong Πάρτε σκορ"
 
 #: src/emu/inpttype.ipp:85
 msgctxt "input-name"
 msgid "P1 Mahjong Double Up"
-msgstr ""
+msgstr "P1 Mahjong Διπλασιασμός"
 
 #: src/emu/inpttype.ipp:86
 msgctxt "input-name"
 msgid "P1 Mahjong Big"
-msgstr ""
+msgstr "P1 Mahjong Μεγάλο"
 
 #: src/emu/inpttype.ipp:87
 msgctxt "input-name"
 msgid "P1 Mahjong Small"
-msgstr ""
+msgstr "P1 Mahjong Μικρό"
 
 #: src/emu/inpttype.ipp:88
 msgctxt "input-name"
 msgid "P1 Mahjong Last Chance"
-msgstr ""
+msgstr "P1 Mahjong Τελευταία Ευκαιρία"
 
 #: src/emu/inpttype.ipp:93
 msgctxt "input-name"
 msgid "P1 Hanafuda A/1"
-msgstr ""
+msgstr "P1 Hanafuda A/1"
 
 #: src/emu/inpttype.ipp:94
 msgctxt "input-name"
 msgid "P1 Hanafuda B/2"
-msgstr ""
+msgstr "P1 Hanafuda B/2"
 
 #: src/emu/inpttype.ipp:95
 msgctxt "input-name"
 msgid "P1 Hanafuda C/3"
-msgstr ""
+msgstr "P1 Hanafuda C/3"
 
 #: src/emu/inpttype.ipp:96
 msgctxt "input-name"
 msgid "P1 Hanafuda D/4"
-msgstr ""
+msgstr "P1 Hanafuda D/4"
 
 #: src/emu/inpttype.ipp:97
 msgctxt "input-name"
 msgid "P1 Hanafuda E/5"
-msgstr ""
+msgstr "P1 Hanafuda E/5"
 
 #: src/emu/inpttype.ipp:98
 msgctxt "input-name"
 msgid "P1 Hanafuda F/6"
-msgstr ""
+msgstr "P1 Hanafuda F/6"
 
 #: src/emu/inpttype.ipp:99
 msgctxt "input-name"
 msgid "P1 Hanafuda G/7"
-msgstr ""
+msgstr "P1 Hanafuda G/7"
 
 #: src/emu/inpttype.ipp:100
 msgctxt "input-name"
 msgid "P1 Hanafuda H/8"
-msgstr ""
+msgstr "P1 Hanafuda H/8"
 
 #: src/emu/inpttype.ipp:101
 msgctxt "input-name"
 msgid "P1 Hanafuda Yes"
-msgstr ""
+msgstr "P1 Hanafuda Ναι"
 
 #: src/emu/inpttype.ipp:102
 msgctxt "input-name"
 msgid "P1 Hanafuda No"
-msgstr ""
+msgstr "P1 Hanafuda Όχι"
 
 #: src/emu/inpttype.ipp:107
 msgctxt "input-name"
 msgid "Key In"
-msgstr ""
+msgstr "Πλήκτρο Μέσα"
```

Comment:
To be fair, “Key In”/“Key Out” don’t make sense in English either – they’re just understood to have certain meanings in the context of gambling machines.  If you asked a random English speaker without relevant experience what they meant, they wouldn’t know.

Another possible way to think of it might be “Deposit Key” for “Key In” and “Withdraw Key” for “Key Out”, since they are key lock switches for depositing and withdrawing credits.